test/tree/fixtures/frequent-deps/paket.lock

```diff
@@ -0,0 +1,66 @@
+NUGET
+  remote: https://www.nuget.org/api/v2
+    A (1.1.1)
+      B (1.1.1)
```

Comment:
looks like this causes a circular dependency. Nuget does not support these. I'm not saying we need to detect circular dependencies, because those should already have been handled by paket, but maybe not have it in our test

Reply:
it's cool we are not crashing though 👍

Reply:
fixed